src/main/java/org/sagebionetworks/bridge/validators/AssessmentValidator.java

```diff
@@ -50,6 +46,9 @@ public void validate(Object target, Errors errors) {
             errors.rejectValue("title", CANNOT_BE_BLANK);   
         }
         validateStringLength(errors, 255, assessment.getTitle(), "title");
+        if (assessment.getPhase() == null) {
+            errors.rejectValue("phase", CANNOT_BE_NULL);
+        }
```

Comment:
What happens to existing assessments?

Reply:
The sql add column migration will set them all to Draft.